src/applications/hca/config/chapters/insuranceInformation/general.js

```diff
@@ -1,53 +1,107 @@
 
+import {
+  healthInsuranceCoverageQuestionDescription,
+  hasTricareWhatIsMyPolicyNumberDescription,
+  healthInsuranceDescription,
+} from '../../../helpers';
+
 const { provider } = fullSchemaHca.definitions;
 const { isCoveredByHealthInsurance } = fullSchemaHca.properties;
 
 export default {
   uiSchema: {
-    'ui:title': 'Other coverage',
+    'ui:description': healthInsuranceDescription,
     isCoveredByHealthInsurance: {
-      'ui:title':
-        'Are you covered by health insurance? (Including coverage through a spouse or another person)',
+      'ui:title': 'Do you have health insurance coverage?',
+      'ui:description': healthInsuranceCoverageQuestionDescription,
       'ui:widget': 'yesNo',
     },
     providers: {
       'ui:options': {
-        itemName: 'Insurance Policy',
         expandUnder: 'isCoveredByHealthInsurance',
+        itemName: 'insurance policy',
+        hideTitle: true,
         viewField: InsuranceProviderView,
       },
       'ui:errorMessages': {
         minItems: 'You need to at least one provider.',
       },
       items: {
         insuranceName: {
-          'ui:title': 'Name of provider',
+          'ui:title': 'Name of insurance provider',
+          'ui:autofocus': true,
         },
         insurancePolicyHolderName: {
-          'ui:title': 'Name of policyholder',
+          'ui:title':
+            'Name of policyholder (the person whose name the policy is in)',
+        },
+        'view:policyOrGroupDesc': {
+          'ui:description': (
+            <div className="vads-u-margin-top--6 vads-u-margin-bottom--2 schemaform-block-title schemaform-block-subtitle">
+              {' '}
+              Provide either your insurance policy number or group code.{' '}
+              <span className="schemaform-required-span vads-u-font-weight--normal">
+                (*Required)
+              </span>
+            </div>
+          ),
+        },
+        'view:hasTricare': {
+          'ui:description': hasTricareWhatIsMyPolicyNumberDescription,
         },
         insurancePolicyNumber: {
-          'ui:title':
-            'Policy number (either this or the group code is required)',
-          'ui:required': (formData, index) =>
-            !get(`providers[${index}].insuranceGroupCode`, formData),
-          'ui:errorMessages': {
-            pattern: 'Please provide a valid policy number.',
-          },
+          'ui:title': (
+            <div>
+              <p className="vads-u-margin-bottom--1">Policy number</p>
+              <p className="vads-u-color--gray vads-u-margin-top--0 vads-u-margin-bottom--0">
+                Either this or the group code is required
+              </p>
+            </div>
+          ),
+        },
+        'view:or': {
+          'ui:description': (
+            <div className="schemaform-block-title schemaform-block-subtitle vads-u-margin-bottom--neg2p5">
+              <strong>or</strong>
```

Comment:
I don't know the context of this page, but in bulleted lists (I know this isn't a bulleted list), the content guidelines say to add a `, <strong>or</strong>` to the end of the line - see https://design.va.gov/content-style-guide/bulleted-lists.

Reply:
Removed the <strong> tag as not needed with classes; this is not a bulleted list.